frontend/components/initiative/Toolbar.vue

```diff
@@ -0,0 +1,76 @@
+<script setup lang="ts">
+import { type InitiativeUserRelationship } from '@/openapi/generated/pacta'
+
+const localePath = useLocalePath()
+const { getMaybeMe } = useSession()
+const { t } = useI18n()
+
+const { isAdmin, maybeMe } = await getMaybeMe()
+const prefix = 'InitiativeToolbar'
+const tt = (key: string) => t(`${prefix}.${key}`)
+
+interface Props {
+  initiativeId: string
+  initiativeUserRelationships: InitiativeUserRelationship[]
+}
+const props = defineProps<Props>()
+
+const isManager = computed(() => {
+  const mm = maybeMe.value
+  return (!!mm && props.initiativeUserRelationships.some(r => r.manager && r.userId === mm.id)) || true
```

Comment:
Lol I'm guessing the `|| true` was for testing? Remove pls

Reply:
Done